apps/cowswap-frontend/src/modules/swap/pure/Row/RowFeeContent/index.tsx

```diff
@@ -19,23 +18,14 @@ export interface RowFeeContentProps extends RowWithShowHelpersProps {
 }
 
 export function RowFeeContent(props: RowFeeContentProps) {
-  const swapZeroFee = useSwapZeroFee()
-  const {
-    showHelpers,
-    includeGasMessage,
-    tooltip,
-    feeToken,
-    feeUsd,
-    fullDisplayFee,
-    feeCurrencySymbol,
-    styleProps = {},
-  } = props
+  const { showHelpers, tooltip, feeToken, feeUsd, fullDisplayFee, feeCurrencySymbol, styleProps = {} } = props
   return (
     <StyledRowBetween {...styleProps}>
       <RowFixed>
         <TextWrapper>
-          {swapZeroFee ? 'Est. fees' : 'Fees'}
-          {includeGasMessage}
+          <FeatureGuard featureFlag="swapZeroFee" defaultContent="Fees">
+            Est. fees
+          </FeatureGuard>
```

Comment:
Nope, keep the old code.
The reason for this is that there's logic on top of the feature flag (a/b testing).

https://github.com/cowprotocol/cowswap/pull/3714/files

Reply:
Damn, really, thanks!
https://github.com/cowprotocol/cowswap/pull/3756/commits/986fe0948b71a1ec204547ce4bf32734ee8efd61

Reply:
Great catch!!